pkgs/by-name/pi/pilot-link/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  pkg-config,
+  darwin,
+  popt,
+  bluezSupport ? stdenv.hostPlatform.isLinux,
+  bluez,
+  readlineSupport ? true,
+  readline,
+  enableConduits ? true,
+  bison,
+  enableLibpng ? true,
+  libpng,
+  enableLibusb ? true,
+  libusb-compat-0_1,
+}:
+
+stdenv.mkDerivation {
+  pname = "pilot-link";
+  version = "unstable-2022-09-26";
```

```suggestion
  version = "0_12_3-unstable-2022-09-26";
```

https://github.com/desrod/pilot-link/tags

---

Then you need to at least add "0-" in front.